types/dompurify/index.d.ts

```diff
@@ -86,6 +86,11 @@ declare namespace DOMPurify {
               }
             | undefined;
         WHOLE_DOCUMENT?: boolean | undefined;
+        CUSTOM_ELEMENT_HANDLING?: {
+            tagNameCheck?: RegExp | ((tagName: string) => boolean) | undefined;
+            attributeNameCheck?: RegExp | ((lcName: string) => boolean) | undefined;
```

Comment:
```suggestion
            tagNameCheck?: RegExp | ((tagName: string) => boolean) | null | undefined;
            attributeNameCheck?: RegExp | ((lcName: string) => boolean) | null | undefined;
```
owner uses nulls (as evaluation support both null/undefined). Types should be aligned with README

Reply:
```js
   * @property {RegExp|Function|null} tagNameCheck one of [null, regexPattern, predicate]. Default: `null` (disallow any custom elements)
   * @property {RegExp|Function|null} attributeNameCheck one of [null, regexPattern, predicate]. Default: `null` (disallow any attributes not on the allow list)
```

Reply:
Good catch, thanks!